webapp/components/Comment/Comment.vue

```diff
@@ -89,9 +69,14 @@ import scrollToAnchor from '~/mixins/scrollToAnchor.js'
 
 export default {
   mixins: [scrollToAnchor],
-  data: function() {
+  data() {
+    const anchor = `commentId-${this.comment.id}`
+    const isTarget = this.routeHash === `#${anchor}`
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-15T22:07:40Z" title="Wednesday, October 16th 2019, 12:07:40 am +02:00">Oct 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -89,9 +69,14 @@ import scrollToAnchor from '~/mixins/scrollToAnchor.js'
 
 export default {
   mixins: [scrollToAnchor],
-  data: function() {
+  data() {
+    const anchor = `commentId-${this.comment.id}`
+    const isTarget = this.$route.hash === `#${anchor}`
```


I would prefer not to have `this.$route` in components. Is it possible to pass this as a prop?